api/src/main/java/javax/config/ConfigValue.java

```diff
@@ -90,10 +93,9 @@
      * default (see {@link #as(Class)} then the provided converter is used instead of the built-in one.
      *
      * @param converter The converter for the target type
-     * @param <N> The target type
      * @return This builder as a typed ConfigValue
      */
-    <N> ConfigValue<N> useConverter(Converter<N> converter);
+    ConfigValue<T> useConverter(Converter<T> converter);
```

Comment:
Why should this converter here? Is this converter not loaded via service loader pattern? What is the use case here?

Reply:
No this is intended for when you do not want to use the default converter but a special handling.